### DISCUSSION TITLE
Access filament without user authentication

### DISCUSSION THREAD

**Discussion by shirokovnv** [filamentphp/filament#6119] [General]
Hello, guys!

Thanks for your effort and awesome package!

My question is: can filament admin panel be accessed without user login/authentication ?

Neither docs nor discussions specifies that.

The first thing came to my mind was the following:

1. make filament-user
2. make custom authentication middleware where set the User from step 1.
3. include it in middleware stack in filament config.

But, is there more convenient way to do such a thing?

Thank you!

**Comment by danharrin** [comment]
Admin panel is not meant to be used by unauthenticated users. You can use the form builder and table builder outside of the admin panel, wherever you want, which is good for guests too.

**Reply by shirokovnv** [reply]
That sounds reasonable.

I believe, we have very specific usecase, when administration tools can be accessed only via corporate network outline. 

That's why we don't need to define any user.

Anyway, thanks for your answer! I will close the discussion, if you dont mind.

**Reply by danharrin** [reply]
You could use a middleware to auto
automatically log users in so they can access it maybe? Or a link that logs them in and redirects.

**Reply by bilogic** [reply]
thanks, apparently using `->login(null)` when registering a panel will disable Filament's built-in auth

**Reply by matthewdanepeavoy** [reply]
Updates to @bilogic. You also need to remove the `Authenticate::class` middleware and `AccountWidget::class` in the AdminPanelProvider. But otherwise it worked!

https://filamentphp.com/docs/3.x/panels/users#setting-up-guest-access-to-a-panel

**Reply by bilogic** [reply]
Any example on how to remove the middleware and widget? Like this? It did not work for me in pest, still getting 403
```
->authMiddleware([])
->widgets([])
->login(null)
```
